libs/blocks/global-navigation/global-navigation.js

```diff
@@ -35,7 +35,7 @@ import { replaceKey, replaceKeyArray } from '../../features/placeholders.js';
 
 const CONFIG = {
   icons: {
-    company: '<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 133.46 118.11" alt="Adobe, Inc."><defs><style>.cls-1{fill:#fa0f00;}</style></defs><polygon class="cls-1" points="84.13 0 133.46 0 133.46 118.11 84.13 0"/><polygon class="cls-1" points="49.37 0 0 0 0 118.11 49.37 0"/><polygon class="cls-1" points="66.75 43.53 98.18 118.11 77.58 118.11 68.18 94.36 45.18 94.36 66.75 43.53"/></svg>',
+    company: '<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 133.5 118.1" alt="Adobe, Inc."><defs><style>.cls-1 {fill: #eb1000;}</style></defs><g><g><polygon class="cls-1" points="84.1 0 133.5 0 133.5 118.1 84.1 0"/><polygon class="cls-1" points="49.4 0 0 0 0 118.1 49.4 0"/><polygon class="cls-1" points="66.7 43.5 98.2 118.1 77.6 118.1 68.2 94.4 45.2 94.4 66.7 43.5"/></g></g></svg>',
```

Comment:
*Used the logo provided in the dotcom ticket.

Reply:
Before: https://main--milo--adobecom.hlx.page/drafts/osahin/prs/dotcom-83993
After: https://dotcom-83993--milo--mokimo.hlx.page/drafts/osahin/prs/dotcom-83993

@overmyheadandbody I copied over the alt text, but for SVGs it seems that a `title` should be used though, any opinion on this? https://www.w3.org/TR/SVG2/struct.html#DescriptionAndTitleElements

Reply:
I'd just suggest to test it out and see what the screen reader (Cmd+F5) announces. I think ideally we want to also have a `role="img"` attribute. If `title` doesn't work as an attribute, it might need to be nested inside the `svg` elements with an ID and then referenced via an `aria-labelledby`

Reply:
The alt text is not used at all, the `aria-labelledby` is being announced which is Adobe, so that makes enough sense and we can remove the faulty alt text on the svg itself

Reply:
For context, the screen reader announces something like "Navigation > link > image > adobe"